omero/sysadmins/windows/server-installation.txt

```diff
@@ -328,120 +278,35 @@ is having a properly configured database.
 
-   #. Double-click on the :guilabel:`PostgreSQL 9.1` database (or right-
+   #. Double-click on the :guilabel:`PostgreSQL` database (or right-
       click and choose :guilabel:`Connect`) and provide your *postgres* user
       login password set during the installation, above.
 
-      .. figure:: /images/install-windows-screenshots/pgadmin-02initialview.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: Connect to the database server
-
-         Connect to the database server
-
-      .. figure::  /images/install-windows-screenshots/pgadmin-03connect.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: Enter password
-
-         Enter password
-
    #. Right-click on :guilabel:`Login Roles` and select :guilabel:`New Login
       Role...`
 
-      .. figure:: /images/install-windows-screenshots/pgadmin-04newrole-context.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: New login role
-
-         New login role
-
-   #. Create a new role and record the name and password. You will need to 
-      configure OMERO to use your username and password by setting the 
+   #. Create a new role and record the name and password. You will need to
+      configure OMERO to use your username and password by setting the
       :property:`omero.db.name` and :property:`omero.db.pass` properties.
 
-      .. warning::
-          For illustrative purposes, the default name and password for the 
-          role are ``db_user``  and ``db_password`` respectively. However, 
-          you should not use these default values for your installation but 
-          use your own choice of username and password instead.
-
-   .. figure:: /images/install-windows-screenshots/pgadmin-05newrole-name.png
-      :figwidth: 65%
-      :width: 60%
-      :align: center
-      :alt: New role name
-
-      Setting name of new login role
-
-   .. figure:: /images/install-windows-screenshots/pgadmin-06newrole-passwd.png
-      :figwidth: 65%
-      :width: 60%
-      :align: center
-      :alt: New role password
-
-      Setting password of new login role
 
 -  Create an ``omero_database`` database:
 
    #. Right-click on :guilabel:`Databases` and select :guilabel:`New Database
       ...`
 
-      .. figure:: /images/install-windows-screenshots/pgadmin-07newdatabase-context.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: New database
-
-         New database
-
    #. Create a new database with the :guilabel:`Name` ``omero_database`` and
       :guilabel:`Owner` ``db_user`` (this may take a few moments)
 
-      .. figure:: /images/install-windows-screenshots/pgadmin-08newdatabase-name.png
-         :figwidth: 65%
-         :width: 60%
-         :align: center
-         :alt: New database name
-
-         New database name
-
 -  Confirm PL/pgSQL language support in your newly created database
```

Comment:
This last section can be removed entirely since it's done by default.  All the way down to and including "Finally, add the plpgsql language, accepting all defaults."